lib/location.ex

```diff
@@ -28,4 +28,10 @@ defmodule Location do
     Logger.debug("Loading location database #{inspect(module)} took: #{time}s")
     :ok
   end
+
+  def version() do
+    version_file = Application.app_dir(:location, "priv/version")
+
+    File.read!(version_file)
```

Comment:
Can it be a function? I'd like to remove most of the runtime behaviour from this library (wanted to discuss it with @ukutaht on the next call) ...